arbitrary-users-patch/check_sidecar_image_digests.sh

```diff
@@ -59,13 +67,17 @@ while read -r line; do
     echo "[INFO] Image ${base_image_name} has valid digest"
   fi
 done < "${SCRIPT_DIR}"/base_images
-mv "${SCRIPT_DIR}"/base_images  "${SCRIPT_DIR}"/base_images.copy
+mv "${SCRIPT_DIR}"/base_images.copy  "${SCRIPT_DIR}"/base_images
 
 set +e
-hasChanges=$(git diff --exit-code "${SCRIPT_DIR}"/base_images)
-if [[ ${hasChanges} -eq 1 ]]; then
-  echo "[INFO] Changes detected, generating PR with new digests"
-  createPR
+if [[ $(git diff --exit-code "${SCRIPT_DIR}"/base_images) ]]; then
+  if [[ ${NO_OP} == "true" ]]; then
+    echo "[INFO] Changes detected, see base_images file changes:"
+    git diff --exit-code "${SCRIPT_DIR}"/base_images
```

Comment:
this means the script will exit with error code 1 if diff found, so the action will fail. Is that what you want here?

Reply:
right, I don't need an exit code here